### ISSUE TITLE
Chapter split not always working

### ISSUE THREAD

**Issue by kenwonders** [openaudible/openaudible#1207]
Sean Lock: 15 Minutes of Misery & 15 Storeys High is splitting into 5 chapters that do not relate to the real chapter order.

Version: 3.7.5. Checked chapter list in get info and it is strangely corrupted.

![image](https://user-images.githubusercontent.com/5612157/235489428-b931c18d-c183-46ee-aec1-b8f753c78a65.png)

Here#s how it looks in Audible player:
![image](https://user-images.githubusercontent.com/5612157/235489753-5dcdbb7f-6a7f-47e6-9383-9dfef4113025.png)

**Comment by openaudible** [comment]
We get the chapter data from either the file downloaded from Audible or from the API. We try to choose the best to one to split with. When you attempt to split, there will be a lot of log messages. Perhaps the log message will be helpful. The version that shows in the Chapter table above is usually from the Audible API.  But when you go to split the book, we also check to see if the meta data in the file contains better chapters. We then run a function called "determineBestChapters" and we output both:

```
                    log.info("Book chapters: \n" + Chapters.instance.inspect(best));
                    log.info("File chapters: \n" + Chapters.instance.inspect(chapters));
```

From there, we're supposed to take the one with more data or better names.

Occasionally the meta data from Audible is not the best.

Sometimes deleting the book, redownloading, and re-converting it will update the file chapter data.

You can send a bug report.. especially if you think we are choosing the wrong chapter data to use.

**Comment by openaudible** [comment]
We just posted a beta with additional debugging and a bug fix to not add missing chapters that are very short in duration.
https://openaudible.org/beta

**Comment by kenwonders** [comment]
Thanks for looking at this. I can clearly see the chapter information in the JSON data, but it seems to have not decoded properly. I'm not sure if the log is showing the raw data or your deserializing so hard to say from my side.

![image](https://user-images.githubusercontent.com/5612157/236643751-baa77fcf-3b43-4c42-83a1-87ab1fd5ef24.png)

The attached files are from the beta.

[log.txt](https://github.com/openaudible/openaudible/files/11413261/log.txt)

[diagnostics.txt](https://github.com/openaudible/openaudible/files/11413259/diagnostics.txt)=

**Comment by openaudible** [comment]
Ok. We found one issue that could have caused this. This may or should be fixed in the latest beta. Please give it a try. OpenAudible.org/beta

**Comment by kenwonders** [comment]
Hi many thanks for looking into it, sadly it hasn't resolved the issue. I deleted books.json and the aax/mp3s and refreshed and it made 5 chapters similar to the initial issue. I've sent a bug report through the app if you're still interested in resolving this.

thanks

**Comment by openaudible** [comment]
Fixed in 3.7.6. Reopen or comment if you notice anything not working. Thank you for the feedback!

**Comment by kenwonders** [comment]
No still not working, same as last bug report.
